ada_feeding/ada_feeding/behaviors/moveit2/servo_move.py

```diff
@@ -42,17 +42,20 @@ def blackboard_inputs(
         pub_topic: Union[BlackboardKey, str] = "~/servo_twist_cmds",
         pub_qos: Union[BlackboardKey, QoSProfile] = QoSProfile(depth=1),
         default_frame_id: Union[BlackboardKey, str] = "world",
+        curr_distance: Optional[BlackboardKey] = None,  # float
     ) -> None:
         """
         Blackboard Inputs
 
         Parameters
         ----------
         twist: Twist or TwistStamped to publish, with updated time stamps.
-        duration: How long to publish the twist
+        duration: How long to publish the twist. If negative, then run forever.
         pub_topic: Where to publish servo TwistStamped messages
         pub_qos: QoS for publisher
         default_frame_id: frame_id to use if Twist type is provided.
+        curr_distance: If None, use time as distance. Otherwise, use this
+            BlackboardKey to store the remaining distance.
```

Comment:
nit: "this BlackboardKey stores..." (not "use" since this node doesn't write to it.)